project/src/components/card-list/card-list.tsx

```diff
@@ -1,9 +1,33 @@
 import Card from '../card/card';
+import {Offer} from '../../types/offer';
+import {useState} from 'react';
+
+type CardListProps = {
+  offers: Offer[];
+}
+
+function CardList ({offers}: CardListProps): JSX.Element {
+  const [offersId, setOffersId] = useState(0);
```

Comment:
Пока можно сделать так раз нам offersId не нужен
```suggestion
  const [, setOffersId] = useState(0);
```